src/pages/components/tiles.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Tiles
+description: >
+  Tiles are Radio buttons styled with icons or images.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/tiles/components/Tile.tsx
```

Comment:
Unfortunately, this needs to be removed as it indicates an `import` under "Configuration" that is invalid for the package.

Reply:
Totally blanked on that consideration. 💯

Reply:
Noting the example addition in the punt list doc.